advisor/src/main/kotlin/advisors/VulnerableCode.kt

```diff
@@ -116,4 +116,18 @@ class VulnerableCode(name: String, vulnerableCodeConfiguration: VulnerableCodeCo
         if (scores.isEmpty()) return listOf(VulnerabilityReference(sourceUri, null, null))
         return scores.map { VulnerabilityReference(sourceUri, it.scoringSystem, it.value) }
     }
+
+    /**
+     * Return a meaningful identifier for this vulnerability that can be used in reports. Obtain this identifier from
+     * the defined aliases if there are any. The data model of VulnerableCode supports multiple aliases while ORT's
+     * [Vulnerability] has just one identifier. To resolve this discrepancy, prefer CVEs over other identifiers. If
+     * there are no aliases referencing CVEs, use an arbitrary alias, assuming that every alias is preferable over
+     * the internal ID of VulnerableCode. Only if no aliases are defined, use the latter as fallback. Note that it
+     * should still be possible via the references to find the aliases that have been dropped.
+     */
+    private fun VulnerableCodeService.Vulnerability.preferredIdentifier(): String {
```

Comment:
Maybe rename this to `preferredCommonId()` or so to somewhat explain in what *way* this id is "preferred" (in the sense that it is more common that a tool-internal id).

Reply:
Renamed as proposed.